### PR TITLE
add missing localizations

### PR DESCRIPTION
**Pull request by L1cardo**
1. add missing localizations
2. improve Simplified Chinese localization

### CODE REVIEW

File: src/server/server.py [hunk 1]
```diff
@@ -2310,12 +2310,12 @@ def check_bpillfw_file(data):
...
-            fwTimStr = "unknown"
+            fwTimStr = __("unknown")
         fileSize = len(dataStr)
         logger.debug("Node update firmware file size={}, version={}, {}build timestamp: {}".\
                      format(fileSize, fwVerStr, fwTypStr, fwTimStr))
-        infoStr = "Firmware update file size = {}<br>".format(fileSize) + \
-                  "Firmware update version: {} ({}Build timestamp: {})<br><br>".\
+        infoStr = __("Firmware update file size = {} bytes<br>").format(fileSize) + \
+                  __("Firmware update version: {} ({}Build timestamp: {})<br><br>").\
                   format(fwVerStr, fwTypStr, fwTimStr)
```
Comment (ethomas997) [inline src/server/server.py:2319]: The non-text parts need to be separated; for instance like this:
```
       infoStr = "{} = {} {}<br>".format(__("Firmware update file size"), fileSize, __("bytes")) + \
                  {}: {} ({}{}: {})<br><br>").\
                  format(__("Firmware update version"), fwVerStr, fwTypStr, __("Build timestamp"), fwTimStr)
```
Similar for several places below.

**Comment by L1cardo** [comment]
@ethomas997 Fixed

**Comment by HazardCreative** [comment]
Wouldn't it be better to have everything in the string, so that the order can be determined by localization? And to move structural parts out of the localization?
`__("Firmware update version: {0} ({1}Build timestamp: {2})").format(fwVerStr, fwTypStr, fwTimStr) + "<br><br>"`

**Comment by L1cardo** [comment]
> Wouldn't it be better to have everything in the string, so that the order can be determined by localization? And to move structural parts out of the localization? `__("Firmware update version: {0} ({1}Build timestamp: {2})").format(fwVerStr, fwTypStr, fwTimStr) + "<br><br>"`

I have considered that, above is not actually a sentence, so it won't have language order issue. It is just something like version num.
For other strings that are sentences, I did what you mean.

**Comment by ethomas997** [comment]
Looks good

**Pull request merged**